src/main/kotlin/no/digdir/accessrequestapi/controller/HandlekurvController.kt

```diff
@@ -0,0 +1,39 @@
+package no.digdir.accessrequestapi.controller
+
+import no.digdir.accessrequestapi.client.FelleskatalogClient
+import no.digdir.accessrequestapi.model.Handlekurv
+import org.springframework.beans.factory.annotation.Value
+import org.springframework.http.HttpStatus
+import org.springframework.http.ResponseEntity
+import org.springframework.web.bind.annotation.ExceptionHandler
+import org.springframework.web.bind.annotation.GetMapping
+import org.springframework.web.bind.annotation.PathVariable
+import org.springframework.web.bind.annotation.RequestMapping
+import org.springframework.web.bind.annotation.ResponseStatus
+import org.springframework.web.bind.annotation.RestController
+import org.springframework.web.reactive.function.client.WebClientResponseException
+import java.util.UUID
+
+@RestController
+@RequestMapping(value = ["/handlekurv"], produces = ["application/json"])
+class HandlekurvController(
+    @Value("\${url.fellesdatakatalog.frontend}")
+    val fellesdatakatalogFrontendUrl: String,
```

Comment:
Ikke bruk `@Value`, `@ConfigurationProperties` er mye enklere å jobbe med.

ie
````
@ConfigurationProperties("url.fellesdatakatalog")
data class FdkUrls(
    val api: String,
    val frontend: String
)
````
og da kan du ta den inn her
```suggestion
    val fdkUrls: FdkUrls,
```

Reply:
Nice 🚀 Endrer i https://github.com/fellesdatakatalog/access-request-api/pull/5/commits/94e14a3c76d1f3cad866165f3f2b5f36428656b7